### ISSUE TITLE
`phi` and `varphi` interchanged

### ISSUE THREAD

**Issue by manuel489** [metanorma/metanorma-standoc#335]
The math symbols `phi` and `varphi` are interchanged in Metanorma

````
phi symbol: stem:[phi]
[stem%unnumbered]
++++
phi
++++

varphi symbol: stem:[varphi]
[stem%unnumbered]
++++
varphi
++++
````

*Asciidoc output:*
![asciidoc](https://user-images.githubusercontent.com/33627611/93369956-1d26a280-f81e-11ea-87d3-57cc440c61a8.PNG)

*Metanorma output:*
![metanorma](https://user-images.githubusercontent.com/33627611/93369949-1b5cdf00-f81e-11ea-85ae-7db0a81f1703.PNG)

**Comment by opoudjis** [comment]
Well this is nasty.

Metanorma and AsciiMath treat phi (curly phi) as U+03C6 and phi symbol (stroke phi) as U+03D5. Asciidoctor and MathJax swap them.

This is an old issue: 

* TeX made phi be stroke phi and varphi be curly phi
* Unicode did the same until 1998, and then reversed them
* Metanorma is doing whatever the input method is telling it to do: all *it* sees is Unicode character values
* AsciiMath http://asciimath.org, http://math.chapman.edu/~jipsen/asciimathjax/ is using the TeX mapping
* Asciidoc AsciiMath is using the Unicode mapping
* MathJax has a setting to flip phi and varphi, and LaTeX has a setting to default phi to varphi
* But Metanorma is seeing MathML with Unicode Greek; this issue arises in the AsciiMath processing.
* MathJax and AsciiMathML default to TeX mapping, with an option to turn off "fixphi" and use Unicode mapping: https://github.com/asciimath/asciimathml/issues/14 .
* It seems that Asciidoc AsciiMath  should follow suit, and I am raising a ticket with them.

**Comment by opoudjis** [comment]
Will be included in asciimath 2.0.2, already in asciimath github.
